app/views/partners/show.html.erb

```diff
@@ -33,29 +33,36 @@
           </div>
           <div class="card-body p-0">
             <div class="row">
-              <div class="col-sm-4 col-4">
+              <div class="col-sm-3 col-3">
                 <div class="description-block border-right">
                   <h1 style="color:purple"><%= total_on_hand(@impact_metrics.dig("agency", "families_served") || 0) %> </h1>
                   <h5>Families served</h5>
                 </div>
                 <!-- /.description-block -->
               </div>
               <!-- /.col -->
-              <div class="col-sm-4 col-4">
+              <div class="col-sm-3 col-3">
                 <div class="description-block border-right">
                   <h1 style="color:purple"><%= total_on_hand(@impact_metrics.dig("agency", "children_served") || 0) %> </h1>
                   <h5>Children served</h5>
                 </div>
                 <!-- /.description-block -->
               </div>
               <!-- /.col -->
-              <div class="col-sm-4 col-4">
+              <div class="col-sm-3 col-3">
                 <div class="description-block border-right">
                   <h1 style="color:purple"><%= total_on_hand(@impact_metrics.dig("agency", "family_zipcodes") || 0) %> </h1>
                   <h5>Zipcodes served</h5>
                 </div>
                 <!-- /.description-block -->
               </div>
+              <div class="col-sm-3 col-3">
+                <div class="description-block border-right">
+                  <h3 style="color:purple"><%= total_on_hand(@impact_metrics.dig("agency", "family_zipcodes_list")&.join(", ") || 0) %> </h3>
```

Comment:
```suggestion
                  <h3 style="color:purple"><%= @impact_metrics.dig("agency", "family_zipcodes_list")&.join(", ")  %> </h3>
```
`total_on_hand` would return the number of elements in `family_zipcodes_list` whereas we want to show the list itself.